src/types/onyx/ApprovalWorkflow.ts

```diff
@@ -112,7 +112,7 @@ type ApprovalWorkflowOnyx = Omit<ApprovalWorkflow, 'approvers'> & {
     /**
      * Errors for the workflow
      */
-    errors?: Record<string, TranslationPaths>;
+    errors?: Record<string, TranslationPaths> | null;
```

Comment:
Instead of doing this you can change `setApprovalWorkflow` argument:
```ts
function setApprovalWorkflow(approvalWorkflow: OnyxMergeInput<typeof ONYXKEYS.APPROVAL_WORKFLOW>) {
    Onyx.merge(ONYXKEYS.APPROVAL_WORKFLOW, approvalWorkflow);
}
```